gobblin-metrics/src/main/java/gobblin/metrics/MetricContext.java

```diff
@@ -149,7 +150,7 @@ public void addChildContext(String childContextName, MetricContext childContext)
    * @return Map of child contexts.
```

Comment:
I suggest renaming this to `getChildContextsAsMap`, and updating the Javadoc to `Get a view of the child {@link MetricContext}s as a {@link ImmutableMap}.`. Please also update `@return an {@link ImmutableMap} of child {@link MetricContext}s keyed on their names`.